src/main/java/uk/ac/cam/cl/dtg/isaac/api/GameboardsFacade.java

```diff
@@ -91,6 +93,8 @@
 
     private final FastTrackManger fastTrackManger;
 
+    private static String validIdRegex = "^[a-z0-9_-]+$";
```

Comment:
Could you make this `final` so that it cannot accidentally be changed? And use capitals to denote a constant?
Like `private static final String NUMERIC_QUESTION_PARSE_REGEX` in the Numeric Validator?

Reply:
And make it clear that it's a gameboard ID, `VALID_GAMEBOARD_ID_REGEX`?

Reply:
done!